### DISCUSSION TITLE
IKEA buttons drain battery extremely quickly

### DISCUSSION THREAD

**Discussion by skorokithakis** [arendst/Tasmota#16631] [Q&A]
I have some IKEA buttons and switches, and they go through a battery in a few days, which is crazy, especially for a button that never gets pressed.

Does anyone have an idea about the cause, or a solution?

**Comment by s-hadinger** [comment]
You can play with the EPID, it's called `ExtPanID` in `ZbConfig`. But whenever you change the Extended PanID you may need to pair again all the devices.

**Reply by skorokithakis** [reply]
Ah, thank you! I'll try that, pairing again should be fine.

**Reply by skorokithakis** [reply]
Apparently I had linked to the wrong issue, it was about pairing issues, not battery issues. It's not necessary to change the `ExtPanID` for this, then.
